### ISSUE TITLE
[$30] Time shown on threads

### ISSUE THREAD

**Issue by jmgasper** [topcoder-platform/forums#334]
Reported against prod:

It shows current time for each threads,instead of lastest posted time of them, really hard to follow when the new posted thread in the forum

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30160873 has been created for this ticket.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30160873 has been updated - it has been assigned to obog.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by atelomycterus** [comment]
@jmgasper I found this bug during testing  https://github.com/topcoder-platform/forums/issues/311. See my comment https://github.com/topcoder-platform/forums/issues/311#issuecomment-750997270. I've fixed it.

**Comment by jmgasper** [comment]
@atelomycterus - Ah, sorry, I missed that.  Thanks!

**Comment by jmgasper** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for ghostar via Topcoder X```
